merlin_models/tf/core.py

```diff
@@ -87,10 +92,18 @@ def __getitem__(self, item):
             if len(item.column_names) > 1:
                 raise ValueError("Schema contains more than one column.")
             item = item.column_names[0]
-        return self.named_variables[str(item)]
+        elif isinstance(item, Tags):
+            item = item.value
+        else:
+            item = str(item)
+        return self.named_variables[item]
 
     def get_embedding(self, item):
-        return self.named_variables[f"{str(item)}/embedding"]
+        if isinstance(item, Tags):
+            item = item.value
+        else:
+            item = str(item)
```

Comment:
I wonder if this could be a helper on `Tags` to keep the behavior encapsulated. Something like:
```
class Tags(Enum):
    ...
    @classmethod
    def value(cls, maybe_tag):
        if isinstance(maybe_tag, Tags):
            return item.value
        else:
            return str(item)
```